Add a link to documentation on ocaml.org

Julow opened the pull request:
Beginners couldn't discover the online documentation because `opam.ocaml.org` ranks higher in search engines than `ocaml.org/p`. I propose to add a link to the documentation on ocaml.org.

kit-ty-kate commented:
> I could test this using the Dockerfile but that was unpractical. The CLI doesn't work on my machine because my local repository is too recent (2.2 vs 2.0). Is here a faster test setup than the Dockerfile ?

Yeah sorry, the dockerfile is meant for deployment not development and i don't know either how to build it locally.
I've pushed it to `staging.opam.ocaml.org` for testing, it should be live within the hour

kit-ty-kate commented:
mmh, i think that's a hair too large ^^"""

<img width="1728" height="428" alt="scrn-2025-09-25-15-28-32" src="https://github.com/user-attachments/assets/56526813-594a-4863-aead-00303c6128e1" />

Julow commented:
Thanks !

Can you try with a force-reload ? It's C-S-r on Firefox.

kit-ty-kate commented:
btw you can see it for yourself in staging.opam.ocaml.org, you should be able to live-update the CSS using firefox's dev tools if you need to tweak the centering and things like that

Julow commented:
I fixed the misalignment in Chrome and removed the unintended bold:
<img width="943" height="91" alt="screenshot-20250926-18-48" src="https://github.com/user-attachments/assets/b8f1c329-172f-4d4c-bc0f-95d16697b893" />

What do you think ?

I'm reluctant to turn the link into a button because it's really a link and not a UI element that user can click and stay on the page like the only other button in the area. Also, buttons are really hard to align vertically.

Julow commented:
You're right, it looks more consistent with a button!

I could properly align the buttons on both Firefox and Chrome:
<img width="1075" height="83" alt="screenshot-20250927-12-04" src="https://github.com/user-attachments/assets/9daa814c-4bab-4255-b712-3a66e5a07146" />

kit-ty-kate commented:
Sadly this changed the font of the text outside the boxes
<img width="846" height="132" alt="scrn-2025-09-27-12-59-06" src="https://github.com/user-attachments/assets/340d3ecc-3ec6-42d1-b3f3-3ba38b11a571" />
<img width="906" height="134" alt="scrn-2025-09-27-12-58-07" src="https://github.com/user-attachments/assets/4074d046-8075-4b50-8852-ffe0433782c7" />

Julow commented:
Good catch! I fixed it. It's now almost the same as before, except for a slightly better vertical alignment.
